data/map.txt

```diff
@@ -29311,6 +29570,15 @@ planet Sunracer
 		threshold 5000
 		fleet "Large Syndicate" 10
 
+planet Swiftsong
+	attributes station hai "hai tourism"
+	landscape land/space6
+	description `Swiftsong Station is maintained by the Imo Blep corporation, and is operated as a giant convention center. Most Hai stations have some degree of restricted access either beyond the docking bay or beyond the promenade possessed by all these stations of the same era. Swiftsong contrasts this by being almost completely open.`
+	description `	Tens of thousands of Hai pass through the docks of Swiftsong on an almost daily basis as the continual churn of summits, forums, conventions, symposiums, productions, conferences, weddings, festivals, and the like roll on in an endless cavalcade of life. With Icelake below and Hai-home just one jump away, this station is a major cultural hub that even houses the head office of one of the biggest music labels in Hai space.`
+	spaceport `The spaceport promenade runs around the interior of the station and is divided into the usual two levels, but has four additional ring levels reaching up where the promenade intersects the base of each of the stations main branches. Almost any business or service you could want can be found somewhere in the spaceport, except for that of an outfitter or shipyard, both of which are precluded by the expansion of the docking bay to accommodate the massive volume of daily traffic.`
```

Comment:
```suggestion
	spaceport `The spaceport promenade runs around the interior of the station and is divided into the usual two levels, but has four additional ring levels in which you can find almost any business or service you could want - except for an outfitter or shipyard. Apparently there is simply no room, as the docking bay must take up nearly all the station's exterior space to accommodate its massive volume of daily traffic.`
```

Reply:
This fundamentally changes the layout of the station. Suggested alternative:
```suggestion
	spaceport `The spaceport promenade runs around the interior of the station and is divided into the usual two levels, but has four additional ring levels rising up from the promenade into each of the stations main branches. Almost any business or service you could want can be found here - except for an outfitter or shipyard. Apparently there is simply no room, as the docking bay must take up nearly all the station's exterior space to accommodate its massive volume of daily traffic.`
```

Reply:
I don't understand how my suggestion changes the layout. Can you explain?

Also, assa general comment, nobody cares about the layout of the station. I know you've got it mapped out in your mind, but the player doesn't care. It's not an interesting detail.

Reply:
In the first case you have a 2-level promenade ring, and 4 separate columns of additional floors rising up at different points where the station branches away from the central core.
In your edit the promenade is just 6-levels high all the way around. These stations are all meant to have been cut from pretty much the same cloth so its incongruous.

And yes, the player may not care when they read the description (though I enjoy finding the quirks in descriptions), but it provides a hook if someone wants to write content that occurs on the station, and gives them something specific to work with.

Reply:
But my suggestion simply leaves that up to the player's imagination rather than making it explicit. It doesn't preclude the possibility of what you have in mind. It simply avoids tediously describing the minutae of the layout which I just don't think is useful or interesting information for the player.